### PR TITLE
Doc: Clean up JA3 plugin docs.

### PR DESCRIPTION
**Pull request by SolidWallOfCode**

### CODE REVIEW

File: doc/admin-guide/plugins/ja3_fingerprint.en.rst [hunk 1]
```diff
@@ -22,28 +22,49 @@
...
 
 JA3 Fingerprint Plugin
-*******************
+**********************
 
 Description
 ===========
 
-``JA3 Fingerprint`` calculates JA3 fingerprints for incoming SSL traffic. "JA3 is a method for creating SSL/TLS client fingerprints" by concatenating values in ClientHello packet and MD5 hash the result to produce a 32 character fingerprint. Malwares tend to use the same encryption code/client, which makes it an effective way to detect malicious clients. More info about ja3 is available: https://github.com/salesforce/ja3.
+The JA3 Ffngerprint plugin calculates JA3 fingerprints for incoming SSL traffic. "JA3" is a method
```
Comment (randall) [inline doc/admin-guide/plugins/ja3_fingerprint.en.rst]: Fingerprint

File: doc/admin-guide/plugins/ja3_fingerprint.en.rst [hunk 2]
```diff
@@ -22,28 +22,49 @@
...
+
+      [time] [client IP] [JA3 string] [JA3 hash]
+
+   By default this is not enabled.
 
 Requirement
 =============
-Won't compile against OpenSSL 1.1.0 due to API changes and opaque structures.
+
+This requires openSSL 1.0.1, 1.0.2, or openSSL 1.1.1 or later. openSSL 1.1.0 will not work due to
```
Comment (randall) [inline doc/admin-guide/plugins/ja3_fingerprint.en.rst]: capitalization of OpenSSL (all 3 cases)

**Comment by bryancall** [comment]
[approve ci doc]

**Comment by bryancall** [comment]
[approve ci docs]

File: doc/admin-guide/plugins/ja3_fingerprint.en.rst (hunk 2 above)
Comment (SolidWallOfCode) [reply doc/admin-guide/plugins/ja3_fingerprint.en.rst]: I thought "openSSL" was the correct case.

**Comment by randall** [comment]
[approve ci docs]

File: doc/admin-guide/plugins/ja3_fingerprint.en.rst (hunk 2 above)
Comment (randall) [reply doc/admin-guide/plugins/ja3_fingerprint.en.rst]: https://www.openssl.org has it as OpenSSL

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

**Pull request merged**